### ISSUE TITLE
Add content tree-based routing

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#8]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [x] Administrator
- [ ] Developer

### Needs what

* To set up URL-to-content resolution (routing) in the administration application.
* To retrieve URLs of pages form the database (not the routing table).

### Why

To make the routing more contributor-centric.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #15, #28, #29, [UrlHelperPageExtensions Core port](https://kentico.atlassian.net/browse/PLATFORM-15572)
